Documentation/VSCodeConfiguration.md

```diff
@@ -4,22 +4,59 @@ Visual Studio Code does not work optimally for Serenity development, and there's
 
 The WSL Remote extension allows you to use VSCode in Windows while using the normal WSL workflow. This works surprisingly well, but for code comprehension speed you should put the Serenity directory on your WSL root partition.
 
-## Code comprehension
+- [Visual Studio Code Project Configuration](#visual-studio-code-project-configuration)
+  - [clangd](#clangd)
+    - [Fixing clangd include errors](#fixing-clangd-include-errors)
+  - [Microsoft C/C++ tools](#microsoft-cc-tools)
+    - [DSL syntax highlighting](#dsl-syntax-highlighting)
```

Comment:
this is a separate tool from `Microsoft C/C++ tools`
so putting it in to the top level should be appropriate

Reply:
I hope this is fixed with the changes I've made